tests/unit/backend/layers/common/test_auth0_manager.py

```diff
@@ -10,6 +10,9 @@
 from backend.common.auth0_manager import auth0_management_session
 
 
+# TODO: Fix this! This is currently being skipped because it is failing due to the ddtrace library.
+# See ticket: https://github.com/chanzuckerberg/single-cell-data-portal/issues/5818
+@unittest.skip("Temporarily skipping test_session_refresh to check impact of ddtrace library.")
```

Comment:
Note: this test verifies refreshing auth0 management credentials. It is not something that is easy to test once deployed. It manages refeshing the auth0 management credentials that are used to manage user API keys. We may not realize this is broken until several hours after the deployment when a user tried to generate credentials for the discovery API.

Reply:
@Bento007 - Then are you saying that this test must be fixed before allowing this PR to merge because it is too important and currently we are unclear on how to fix this test with `ddtrace` installed?

Reply:
That note was for posterity. As long as this is fixed in a timely manner and no code related to how we manage the Auth0 management session is changed, this *shouldn't* break. ☠️ Yee be warned!